products/sle/main.pm

```diff
@@ -116,7 +116,7 @@ sub cleanup_needles {
 }
 
 sub is_desktop_module_available {
-    return check_var('SCC_REGISTER', 'installation') || check_var_array('ADDONS', 'all-packages') || check_var_array('WORKAROUND_MODULES', 'desktop');
+    return check_var('SCC_REGISTER', 'installation') || check_var_array('ADDONS', 'all-packages');
```

Comment:
Please, include, patch I've done for original PR.

Reply:
do you have the hash to cherry-pick it

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/8335/commits/1336d5b4ebd07b955a5a02d02fdf128fc3de386d

Reply:
5875fd4b3 fixed